Documentation/Index.rst

```diff
@@ -1,57 +1,70 @@
-.. include:: /Includes.rst.txt
+..  include:: /Includes.rst.txt
 
-.. _start:
-.. _api-overview:
+..  _start:
+..  _api-overview:
 
 ===============
 TYPO3 Explained
 ===============
 
-:Version:
-   |release|
+..  card-grid::
+    :columns: 1
+    :columns-md: 2
+    :gap: 4
+    :class: pb-4
+    :card-height: 100
 
-:Language:
-   en
+    ..  card:: :ref:`API A-Z <api>`
 
-:Author:
-   TYPO3 contributors
+        This chapter describes the most important elements of the API.
```

Comment:
For me "it describes API elements" is too short. I do not know what the API elements are. So maybe provide some examples or add a few further words to it. I have no idea since I am not used to apply API's in my daily life.

Reply:
Well it is "somehow everything" not sure how to explain this in short

Reply:
How about: "This chapter describes the core functionality of TYPO3 including API and framework configuration"?

Reply:
Sounds good to me